docs/Couchbase-batchsource.md

```diff
@@ -0,0 +1,59 @@
+# Couchbase Batch Source
+
+Description
+-----------
+Reads documents from a Couchbase bucket and converts each document into a StructuredRecord with the help
```

Comment:
StructuredRecord is something that only means something to plugin developers and shouldn't be mentioned.

Reads documents from a Couchbase bucket. A filter can be specified to only output documents that meet a specific criteria.

Reply:
Changed.